flatpak/org.eclipse.theia.yaml

```diff
@@ -34,7 +34,21 @@ modules:
       - type: script
         dest-filename: wrapper.sh
         commands:
-          - exec zypak-wrapper /app/share/theia/theia-ide-electron-app.bin --ozone-platform-hint=auto --no-sandbox "$@"
+          - |
+            # Common Electron flags for Wayland
+            ELECTRON_FLAGS="--ozone-platform-hint=auto --no-sandbox"
+            
+            # When running on host (after sandbox escape), zypak is not available
```

Comment:
Did AI suggest that?

The chain of operations is like this:

- Flatpak runs Theia inside the sandbox
- Theia checks whether it should escape it
  - If not, well nothing happens
  - If so, the current process is replaced. `wrapper.sh` is not re-executed!

You don't need extra logic here.

Reply:
I was able to simplify this so `zypak-wrapper` isn't called anymore at all.